crates/store/src/state.rs

```diff
@@ -49,6 +49,7 @@ struct InnerState {
     nullifier_tree: NullifierTree,
     chain_mmr: Mmr,
     account_tree: SimpleSmt<ACCOUNT_TREE_DEPTH>,
+    latest_block_num: BlockNumber,
```

Comment:
I think we can get the same info from `chain_mmr` - so, `latest_block_num` may not be necessary. That is, we can add a method `InnerState::latest_block_num()` but this method would get this info from `chain_mmr`.

Reply:
Good point, thank you!